data/hai/hai weapons.txt

```diff
@@ -48,9 +48,9 @@ outfit "Charge Gas Sprayer"
 		"reload" 1
 		"firing energy" 2.1
 		"firing heat" 1.2
-		"shield damage" -0.4
-		"ion damage" -0.1
-	description `Charge Gas was developed by the Hai during their war with the Korath as an inexpensive way to force Korath raiding ships into unfavourable combat conditions. The basic principle is simple: an easily producible, stable gas that holds an ionizing charge for long periods of time, deployed in broad sweeps to inhibit enemy movement. When dispensed from a fast moving platform like a Lightning Bug, it becomes a very effective crowd-control tool.`
+		"shield damage" 0.4
```

Comment:
Definitely do not change actual weapon mechanics. The negative damages here are to reduce point-blank damages.